test/repository_test.rb

```diff
@@ -170,9 +73,23 @@
             UserRepository.delete(@persisted_user)
           end
 
-          it 'does not touch created_at' do
-            UserRepository.create(@persisted_user)
-            @persisted_user.created_at.wont_be_nil
+          it 'raises an exception' do
+            -> { UserRepository.create(@persisted_user) }.must_raise Exception
```

Comment:
@aderyabin `Exception` is generic, can you please test against the specific type? Thanks!